tests/suite/fixtures.py

```diff
@@ -785,7 +790,11 @@ def fin():
 
     request.addfinalizer(fin)
 
-    return TransportServerSetup(ts_resource['metadata']['name'], test_namespace)
+    ingress_name = "nginx-ingress"
```

Comment:
ingress_name? or ic_name?

Reply:
I'm not sure of the difference, luckily it's not needed anymore so I removed it!